toolboxes/feel/feelmodels/modelcore/modelnumerical.cpp

```diff
@@ -47,6 +47,7 @@ ModelNumerical::ModelNumerical( std::string const& _theprefix, worldcomm_ptr_t c
         M_timeInitial( doption(_name="ts.time-initial") ),
         M_timeFinal( doption(_name="ts.time-final") ),
         M_timeStep( doption(_name="ts.time-step") ),
+        M_timeOrder( ioption(_prefix=_theprefix, _name="ts.order") ),
```

Comment:
@metivett There is no reason to have time order here!

Reply:
@vincentchabannes Why not ? I think we need an "abstract" way to give the order to any model. Otherwise, the time order is only given by the BDF object, which itself reads it directly from the options, and cannot be set from the code API directly. This is a bit in conflict with the OOP approach...
In particular, I need this for `LevelSet`'s sub-advections, which must use the same time-order as the level-set advection.

Reply:
it's too specific, you can add a virtual function which return 1 by default, but it's require to update all toolboxes. For example, with Newmark or theta scheme, the time order depends of the parameter given, the order is not just one parameter. It's your multiphysics model which should impose the compatibility of your several time scheme

Reply:
Yes, I agree it's too restrictive, but I don't have a better API for the moment... In practice, I think we really need to be able to tune the time-stepping properties dynamically from within the toolboxes.

Reply:
but you can,  what is the problem?

Reply:
You mean, using `bdf->setOrder` ?

Reply:
nope, I thought that you can store the member M_timeOrder inside the advection class. The thing that I don't like is timeOrder, I would prefer to have M_bdfTimeOrder and avoid confusion. Anyway, it's probably the time to have another base class which manages time-stepping properly.